docs/config-file-ort-yml.md

```diff
@@ -127,11 +127,11 @@ scopes defined in the examples below match the scopes in your project.
 
 ### When to Use Curations
 
-Curations should be used when you want to correct the licenses detected in the source code of the
-project. If you need to correct the license findings for a third-party dependency then add a curation to
-[curations.yml](config-file-curations-yml.md) or [package configuration](config-file-package-configuration-yml.md).
+License finding curations should be used when you want to correct the licenses detected in the source code of the
+project. To define path excludes on a global level for third-party packages, please use the
+[package configurations](config-file-package-configuration-yml.md).
```

Comment:
Why is the second sentence explaining  path excludes? e.g. the previous section is about excludes.
I'm not saying it's wrong, just thinking out loud: Should this section only be above the types of curations which can be put into the ort.yml as this file is about the ort.yml?

If yes, then this section should explain (1) license finding curations (2) package curations.

I'd also be fine to leave it as is in this PR, but make a follow up PR to extract the "When to Use Curations" out of this file and move it to a more general place.  The concept of "package curations", "license finding curations" and "path excludes" is  or gets now implemented both 'project-local' and 'global'. The answer to the when to use question is same no matter if "local" or "global" so it shouldn't be in this documentation about "local" configuration.

What do you think?

Reply:
> I'd also be fine to merge this PR as-is, but make a follow up PR to extract the "When to Use Curations" out of this file and move it to a more general place.

As this documentation will be rewritten again as soon as _project-local_ package configurations are made possible, I'd prefer to do this in a separate PR.